site/docs/hosting/vps.md

```diff
@@ -7,26 +7,126 @@ next: false
 
 # Hosting: VPS
 
-A virtual private server, mostly known as VPS, is a virtual machine running in the cloud with its users having the full control of its system.
+A virtual private server, mostly known as VPS, is a virtual machine running in the cloud with its users having full control over its system.
 
-In this guide, you'll learn about various methods of running your bot in a VPS, keeping it online 24/7, making it run automatically when your VPS boots and restart on crashes.
+## Server Rental
 
-## systemd
+> To be able to follow this guide, you first need to rent a VPS.
+> This first section will explain how to do that.
+> If you already have a VPS to work on, skip to the [next section](#starting-the-bot).
+
+We choose [Hostinger](https://hostinger.com) as our host.
+
+> You are free to choose the provider of your choice.
+> All providers provide the same services, so you won't have any problems with the technical part of this article.
+> You can take this part as an overview of how the server rental operation works.
+> If you are a beginner, you can use this guide to rent your first ever server!
+
+::: tip Analog of a Server
+If you cannot or do not want to rent a server but you still want to play around with running a bot on a VPS, you can follow this tutorial on a virtual machine instead.
+To do this, use an application such as [VirtualBox](https://virtualbox.org).
+Create a virtual machine with the desired Linux distribution to simulate a Linux server.
+:::
+
+Go to the [VPS Hosting page](https://hostinger.com/vps-hosting).
+We will use the "KVM 1" plan.
+The resources of "KVM 1" are enough for bots with a large audience, and even more so for our test bot.
+
+Go back to the "KVM 1" card and click the "Add to cart" button.
```

Comment:
```suggestion
Click the "Add to cart" button.
```

I missed the point when I deleted the paragraph about the comparison table. Now this line is not relevant.